src/clientLogger.js

```diff
@@ -12,50 +14,67 @@ import {toJSON} from './_json'
  * })
  * await client.ping() // Logs the request and response.
  */
-export function logger(loggerFunction) {
-  return requestResult => {
-    loggerFunction(showRequestResult(requestResult))
-  }
+function logger(loggerFunction) {
+  return function(requestResult) {
+    return loggerFunction(showRequestResult(requestResult));
+  };
 }
 
 /** Translates a {@link RequestResult} to a string suitable for logging. */
-export function showRequestResult(requestResult) {
-  const {query, method, path, auth, requestContent, responseHeaders, responseContent,
-    statusCode, timeTaken} = requestResult
+function showRequestResult(requestResult) {
+  var query = requestResult.query,
+    method = requestResult.method,
+    path = requestResult.path,
+    auth = requestResult.auth,
+    requestContent = requestResult.requestContent,
+    responseHeaders = requestResult.responseHeaders,
+    responseContent = requestResult.responseContent,
+    statusCode = requestResult.statusCode,
+    timeTaken = requestResult.timeTaken;
 
-  let out = ''
+  var out = '';
+  
   function log(str) {
-    out = out + str
+    out = out + str;
   }
 
-  function indent(str) {
-    const indentStr = '  '
-    return str.split('\n').join('\n' + indentStr)
+  log('Fauna ' + method + ' /' + path + _queryString(query) + '\n');
+  log('  Credentials: ' + (auth == null ? 'null' : (auth.user + ':' + auth.pass)) + '\n');
```

Comment:
i remove the auth.pass completely later, but it still emits the secret. I'll change that behavior.